gnd/src/main/java/com/google/android/gnd/rx/annotations/Events.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.rx.annotations;
+
+import static java.lang.annotation.ElementType.TYPE_USE;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Target;
+
+/**
+ * Denotes an observable that emits events.
+ *
+ * <p>Recommended base types:
+ *
+ * <ul>
+ *   <li>{@link io.reactivex.Observable}
```

Comment:
Since `Observables` are cold by default, should we also mention `Subject` and/or `ConnectableObservable`, or another class that's "hot" by default?

Reply:
Moved this into our doc so we can iterate there instead.